### ISSUE TITLE
(Neo)mutt conflict in Debian Stretch?

### ISSUE THREAD

**Issue by christopherisnow** [LukeSmithxyz/mutt-wizard#417]
I'm running Debian Stretch and was interested in setting up Neomutt with mw. When I get to the end of the wizard I get this error:

```
Log-on not successful.
It seems that either you inputted the wrong password or server settings, or there are other requirements for your account out of the control of mutt-wizard.
```
 
I've tried with a Gmail account and another IMAP account, double-checking the passwords and server settings, with the same results. 

Perhaps this could be to blame: Neomutt is unavailable in Stretch, so I have vanilla Mutt installed, which "is built with the NeoMutt patchset". Is that the likely culprit? If so, is there a workaround?

**Comment by christopherisnow** [comment]
Update: after finally succeeding to build Neomutt from source, I still get the same error.

As a workaround I added basic account info manually to the muttrc, which at least gets my account up but conflicts with mw norms. Copying and pasting that in the wizard never gets me anywhere.

What am I missing?

**Comment by christopherisnow** [comment]
Update 2: a faulty installation of neomutt was to blame. I probably didn't configure the installation from source properly. After installing neomutt from the Debian Buster repos on a live OS, however, everything worked swimmingly.

Debian Stretch users might consider upgrading to Buster before setting up neomutt with mw, or be totally in command of the config options when installing neomutt to begin with, which I was not.

**Comment by KebabLord** [comment]
i also lived a similiar problem, reconfiguring mbsnyc solved the problem #509
